00-max.js

```diff
@@ -1,3 +1,11 @@
 export function max(array) {
-    // TODO: implement max function
+    let output;
+    let isArrayHaveContent = checkArrayContent(array);
+    isArrayHaveContent == true ?  output = Math.max.apply(Math.max,array) :  output = undefined;
```

Comment:
That's a nice use of the `.apply(` method, but in newer versions of javascript, you can just write this instead:
```js
output = Math.max(...array)
```